hack/.golint_failures

```diff
@@ -111,6 +111,7 @@ pkg/cloudprovider/providers/photon
 pkg/cloudprovider/providers/vsphere
 pkg/cloudprovider/providers/vsphere/vclib
 pkg/controller
+pkg/controller/apis/config/v1alpha1
```

Comment:
I would be surprised if `staging/src/k8s.io/kube-controller-manager/config/v1alpha1` wouldn't need to be added,  I think you need to do that to get verify passing

Reply:
no need, we had add comments for all `function` and `var` in  `staging/src/k8s.io/kube-controller-manager/config/v1alpha1`, and the SetDefault_FOO  had remove to `pkg/controller/apis/config/v1alpha1`